package.json

```diff
@@ -12,8 +12,8 @@
         "format": "npm run format:base -- --write",
         "format:base": "prettier .",
         "format:check": "npm run format:base -- --check",
-        "lint": "eslint --ext .ts,.vue --ignore-path .gitignore src",
-        "lint:fix": "eslint --ext .ts,.vue --ignore-path .gitignore src --fix",
+        "lint": "eslint --ignore-path .gitignore src",
+        "lint:fix": "npm run lint -- --fix",
```

Comment:
is there a `--` too much?

Reply:
` -- --fix` basically means: pass down `--fix` to the command the command itself is calling. So it passes down the arguments to the next one.